packages/terra-application-utility/CHANGELOG.md

```diff
@@ -3,6 +3,9 @@ Changelog
 
 Unreleased
 ----------
+### Added
+* Role property added to utility menu
```

Comment:
whoops, lets update this to be menuRole

Reply:
[Updated](https://github.com/cerner/terra-framework/pull/501/commits/36eb776f7973f67fd6cda910594a53aaadcaf71f)